### DISCUSSION TITLE
v0.1.1

### DISCUSSION THREAD

**Discussion by bobbyy16** [bobbyy16/Hyper-List#9] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/bobbyy16/Hyper-List/releases/tag/'>v0.1.1</a>.</em>

**Comment by AydiosExecutive** [comment]
I cant find the dataset of companies along with their career portal links, is there no database retreival. We can use open datasets given by mca or data.gov.in

**Comment by bobbyy16** [comment]
Data is in the local file go to src/DB/company.json. 
Can you elaborate how do you retrieve the data from data.gov.in
